bin/document_slug_updater

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env ruby
+
+require File.expand_path("../../config/environment", __FILE__)
+require './app/lib/specialist_publisher.rb'
+
+def strip_leading_slash(path)
+  path.sub(%r{^/}, '')
+end
+
+slug     = strip_leading_slash(ARGV.fetch(0))
+new_slug = strip_leading_slash(ARGV.fetch(1))
+
+edition = SpecialistDocumentEdition.published.where(slug: slug).last
```

Comment:
We added the `.published` scope as it seemed sensible - it wasn't in the Opsmanual instructions.